src/constants/model.ts

```diff
@@ -55,6 +57,12 @@ export const Model = {
     JUMPER: 'Jumper',
     JUMPY: 'Jumpy'
   },
+  [Make.CUPRA]: {
+    ATECA: 'Ateca',
+    FORMENTOR: 'Formentor',
+    LEON: 'Leon',
+    BORN: 'Born'
```

Comment:
Seems like there is a new CUPRA called `Tavascan`. Let's get that added now as well.

Reply:
I was having issues with the type for the test so I modified it a bit to make sure it was resolving CUPRA vehicles appropriately. [This small change](https://github.com/connectedcars/node-vinutils/pull/83/files#diff-04c722fec454c785bfa483e2b9398b02c607b3cc4d33c0323d35c90fbeb8cf38R697-R699)